Fitness/Models/ProBio.swift

```diff
@@ -0,0 +1,101 @@
+        let routine1 = ProRoutine(title: "Relax and reflect", routineType: .mindfullness, text: "I am an avid knitter and I do yoga regularly. Find something that relaxes you that isn’t physically taxing and turn it into a habit! Additionally, take time to reflect at the beginning or end of each day, actively watching thoughts pass through your head without dwelling on them.")
+        let routine2 = ProRoutine(title: "Go for a run", routineType: .cardio, text: "Start out with small, achievable goals. Start by running up to a mile at a time every time you run, and don’t increase by more than a mile each week. Listen to your body, get the right shoes, and make sure someone checks your form!")
+        let routine3 = ProRoutine(title: "Weightlifting", routineType: .strength, text: "First of all, form is EVERYTHING. Work with someone who can spot you for the movements as you start. Muscle fatigue each time is not desired, so start by doing small rep/low weight circuits of simple movements (back squats, presses, deadlifts) 3x a week, increasing weight 5-10lbs each week.")
+        let proCleo = ProBio(id: "test", name: "Cleo Kyriakides", profilePic: "cleo", secondaryProfilePic: "cleoSecondary", bio: "I’m a long distance runner who got hooked onto CrossFit along the way! I love feeling strong and sharing my knowledge to help people reach their goals. I’m also the 2018-19 Fitness Club prez and a member of Track Club!", links: urls, expertise: ["Running", "Olympic lifting", "CrossFit"], routines: [routine1, routine2, routine3], summary: "Resistance Training and Cardio")
+        return proCleo
+    }
+    
+    public static func getMark() -> ProBio {
+        var urls: [PersonalLink] = []
+        urls.append(PersonalLink(url: URL(string: "https://www.facebook.com/mark.rittiboon"), site: .facebook))
+        let routine1 = ProRoutine(title: "Cook", routineType: .mindfullness, text: "I love to cook! Nothing like Spotify's \"cooking music\" playlist and eggs in the morning :)")
+        let routine2 = ProRoutine(title: "Go for a swim", routineType: .cardio, text: "I don't enjoy running and Ithaca is really cold so I don't do it haha... I find swimming to be really fun and effective. For me, it's also the best form of active recovery!")
+        let routine3 = ProRoutine(title: "Weightlifting", routineType: .strength, text: "I think everyone looking to build strength should focus on compound lifts (Squats, Deadlifts, Bench, etc). A 5x5 beginner program is a good start!")
+        let proMark = ProBio(id: "test", name: "Cleo Kyriakides", profilePic: "mark", secondaryProfilePic: "markSecondary", bio: "I used to be a competitive strength athlete participating in powerlifting and strongman competitions. However, I've transitioned into combat sports ~ recently completing a fighting camp in Phuket, Thailand!", links: urls, expertise: ["Weightlifting"], routines: [routine1, routine2, routine3], summary: "Weightlifting and Combat Sports")
+        return proMark
```

Comment:
are we not getting all this data from backend?

Reply:
These are temporarily hardcoded until I integrate with backend